### PR TITLE
standardize and clean up tests

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

**Comment by tmatsuo** [comment]
Can you summarize 1) changes and 2) how did you make those changes ? Should I really review all the 98 files?

**Comment by bshaffer** [comment]
@tmatsuo thanks! yes, good idea.

1. Rename every `phpunit.xml` file to `phpunit.xml.dist`.
2. Add Google copyright header to every `phpunit.xml.dist`.
3. Indent every `phpunit.xml.dist` file to 4 spaces
4. Add `<testsuite name="...">` to every `phpunit.xml.dist`
5. If `test/bootstrap.php` was only including `vendor/autoload.php`, remove `test/bootstrap.php` and specify `vendor/autoload.php` as the bootstrap file in `phpunit.xml.dist` (see #373)
